.github/workflows/copy.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       - name: Copy dist items into Repo Classifai
         uses: andstor/copycat-action@v3
         with:
-          commit_message: Update files from Repo Classifai_FrontEnd, branch ${{ steps.vars.outputs.branch }} at commit ${{ steps.vars.outputs.sha_short }} # Commit message at repo classifai
+          commit_message: ':lipstick: Update files from Repo Classifai_FrontEnd, branch ${{ steps.vars.outputs.branch }} at commit ${{ steps.vars.outputs.sha_short }}' # Commit message at repo classifai
```

Comment:
double check if the single quote before & after is intended
@fariskamaludin-skymind

Reply:
works as intended. I tested it out

Reply:
thanks @fariskamaludin-skymind